web/src/admin/groups/MemberSelectModal.ts

```diff
@@ -7,13 +7,24 @@ import { TableColumn } from "@goauthentik/elements/table/Table";
 import { TableModal } from "@goauthentik/elements/table/TableModal";
 
 import { msg } from "@lit/localize";
-import { TemplateResult, html } from "lit";
+import { TemplateResult, css, html } from "lit";
 import { customElement, property } from "lit/decorators.js";
 
 import { CoreApi, User } from "@goauthentik/api";
 
 @customElement("ak-group-member-select-table")
 export class MemberSelectTable extends TableModal<User> {
+    static get styles() {
+        return [
+            ...super.styles,
+            css`
+                .show-disabled-toggle-group {
+                    margin-left: 0.5rem;
```

Comment:
Nit: Makes mobile overrides easier to adjust but this is super minor.
```suggestion
                    margin-inline-start: 0.5rem;
```